presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcClient.java

```diff
@@ -85,6 +88,11 @@ PreparedStatement buildSql(ConnectorSession session, Connection connection, Jdbc
 
     boolean isLimitGuaranteed(ConnectorSession session);
 
+    default void setColumnComment(JdbcIdentity identity, JdbcTableHandle handle, JdbcColumnHandle column, Optional<String> comment)
+    {
+        throw new PrestoException(NOT_SUPPORTED, "This connector does not support setting column comments");
```

Comment:
Can you please define issues to cover other JDBC connectors?

Reply:
Filed https://github.com/prestosql/presto/issues/5333